docs/en/sql-reference/data-types/newjson.md

```diff
@@ -670,6 +670,15 @@ SELECT json FROM format(TSV, 'json JSON(a.b.c UInt32, SKIP a.b.d, SKIP REGEXP \'
 The `JSON` data type can store only a limited number of paths as separate sub-columns internally. 
 By default, this limit is `1024`, but you can change it in the type declaration using parameter `max_dynamic_paths`.
 
+:::tip Finding the number of dynamic paths used
+To find the number of dynamic paths in use for a JSON column, you can write:
+
+```
+SELECT length(distinctJSONPaths(json)) FROM test_json;
+```
+
+:::
```

Comment:
That's not right. `distinctJSONPaths` returns all paths, not only dynamic paths. For dynamic paths we don't have aggregate function, only usual function `JSONDynamicPaths`. Also set of dynamic paths can be different in different data parts.

So this tip will be not really useful and can be confusing.

And we have a section about introspection functions to inspect the content of JSON column on this page: https://clickhouse.com/docs/sql-reference/data-types/newjson#introspection-functions. I think it should be enough.

Let's just add a reference to this section here (better to do it after the next paragraph about shared data)